spec/glotpress_downloader_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'spec_helper'
+
+describe Fastlane::Helper::GlotpressDownloader do
+  describe 'downloading' do
+    context 'when auto retry is enabled' do
+      context 'when GlotPress returs a 200 code' do
+        it 'returns the response, without retrying' do
+          downloader = described_class.new(auto_retry: true)
+          fake_url = 'https://test.com'
```

Comment:
`fake_url` is misleading... `dummy_url` might be better?

Reply:
agreed